eventuals/generator.h

```diff
@@ -472,6 +472,11 @@ struct _Generator final {
           std::move(dispatch_));
     }
 
+    template <typename Downstream>
+    static constexpr bool CanCompose = true;
+
+    using Expects = StreamOrValue;
```

Comment:
Unless I'm missing something this is:
```suggestion
    using Expects = SingleValue;
```